openedx/core/djangoapps/appsembler/sites/management/commands/save_all_sites.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+
+
+class Command(BaseCommand):
+    help = "Force save on all sites"
+
+    def handle(self, *args, **options):
+        if not settings.ROOT_URLCONF == 'lms.urls':
+            raise CommandError('This command can only be run with the lms settings')
```

Comment:
Thanks, one minor amendment though:

```suggestion
            raise CommandError('This command can only be run from within the LMS')
```